master/static/srv/get_models.sql

```diff
@@ -1,10 +1,10 @@
-SELECT m.id, m.name, m.description, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels, u.username, m.archived, COUNT(mv.version) as num_versions FROM models as m
+SELECT m.id, m.name, m.description, m.notes, m.metadata, m.creation_time, m.last_updated_time, array_to_json(m.labels) AS labels, u.username, m.archived, COUNT(mv.version) as num_versions FROM models as m
 LEFT JOIN model_versions as mv ON mv.model_id = m.id
 LEFT JOIN users as u ON u.id = m.user_id
 WHERE ($1 = '' OR m.archived = $1::BOOL)
 AND ($2 = '' OR (u.username IN (SELECT unnest(string_to_array($2, ',')))))
 AND ($3 = '' OR (m.labels <@ string_to_array($3, ',')))
-AND ($4 = '' OR LOWER(m.name) LIKE $4)
-AND ($5 = '' OR LOWER(m.description) LIKE $5)
+AND ($4 = '' OR LOWER(m.name) = $4)
+AND ($5 = '' OR (LOWER(m.description) LIKE CONCAT($6::text, $5, $6::text)))
```

Comment:
the only time you're using $6, it's `%`. I don't really see how $6 can be any other value in a useful way. 
could this just be hardcoded to `ILIKE ('%%' || $5 || '%%')`?

Reply:
come to think of it, postgres has `ILIKE` as case-insensitive LIKE. so we could 
- skip lower here
- not call ToLower in the go code for the input parameter.

Reply:
oh this was only an issue where the SQL builder didn't know what to do with '%', if I can do %% then that's better